web/src/app/pages/edit-survey/edit-details/edit-details.component.html

```diff
@@ -14,9 +14,27 @@
   limitations under the License.
 -->
 
-<survey-details
-  #surveyDetails
-  [title]="survey?.title"
-  [description]="survey?.description"
-  (onValueChanges)="onDetailChanges($event)"
-></survey-details>
+<div class="survey-details-container">
+  <div class="survey-detail-card">
+    <div class="card-title">Survey name and description</div>
+
+    <div class="card-row">
+      <survey-details
+        #surveyDetails
+        [title]="survey?.title"
+        [description]="survey?.description"
+        (onValueChanges)="onDetailChanges($event)"
+      ></survey-details>
+    </div>
+  </div>
+
+  <div class="survey-detail-card">
+    <div class="card-title">Delete survey</div>
+    <div class="card-row delete-survey">
+      <div style="flex: 1;">Permanently delete survey and associated data</div>
+
+      <button mat-flat-button (click)="openDeleteSurveyDialog()">
```

Comment:
In the video the background looks pink, could just be my eyes are off. Also please check the spacing:

<img width="211" alt="image" src="https://github.com/google/ground-platform/assets/228050/4011e5b8-75fa-47bb-b6c4-77519dd6c41a">